setup.py

```diff
@@ -16,11 +16,6 @@
         long_description = f.read()
 
 
-if sys.version_info < (3, 2):
```

Comment:
And ideas why use this old way will install `futures` on python versions higher than 3.2?